src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -144,7 +144,33 @@ private void findAndStartLsForFile(@NotNull VirtualFile file,
         getLanguageServers(file, null, definition);
     }
 
+    /**
+     * Returns true if the given file matches one of started language server with the given filter and false otherwise.
+     * @param file the file.
+     * @param filter the filter.
+     * @return true if the given file matches one of started language server with the given filter and false otherwise.
+     */
     @NotNull
+    public boolean isMatchStartedLanguageServers(@NotNull VirtualFile file,

```

Comment:
I'd rather avoid having this ultra-specific public method here.

Might as well have a more generic `hasAny(servers, filter)`  and/or `find(servers, filter)` methods in a utility class

Reply:
I use now this method in LSP formatting.